app/src/components/map.jsx

```diff
@@ -290,9 +158,26 @@ class Map extends Component {
     if (!nextProps.map) {
       return;
     }
-    this.updateLayersState(nextProps);
-    this.updateFiltersState(nextProps);
-    this.updateTrackLayer(nextProps);
+
+    const newInnerExtent = nextProps.filters.timelineInnerExtent;
+    if (extentChanged(newInnerExtent, this.props.filters.timelineInnerExtent)) {
+      this.state.overlay.drawTimeRange(newInnerExtent[0].getTime(), newInnerExtent[1].getTime());
+      // this.setState({ currentTimestamp: startDate });
+      // this.state.overlay.updateFilters(this.props.filters);
+      // paint track layer
+      // if (this.state.trackLayer) {
+      //   this.state.trackLayer.drawTile(
+      //     this.props.map.track.seriesGroupData,
+      //     this.props.map.track.selectedSeries,
+      //     this.props.filters,
+      //     this.state.currentTimestamp || this.props.filters.startDate
+      //   );
+      // }
+    } else {
```

Comment:
In theory no, because `props.filters.timelineInnerExtent` is a prop I added separately. It only changes when the user interacts with the new timeline, therefore what used to be triggered for all props updates, will be triggered for all props updates _except_ the new one.